web/styles/left_sidebar.css

```diff
@@ -372,6 +380,12 @@ ul.filters {
                     opacity: 0.75;
                 }
             }
+
+            .topic-list
+                li:not(.has_unmute_unread, .unmuted_topic)
```

Comment:
Why is this the selector in use here? It seems complicated and I'm not sure why it'd be correct.

Reply:
removed this selector and made CSS simpler